pandas/core/window/common.py

```diff
@@ -336,3 +315,12 @@ def _prep_binary(arg1, arg2):
     Y = arg2 + 0 * arg1
 
     return X, Y
+
+
+def get_weighted_roll_func(cfunc: Callable) -> Callable:
```

Comment:
Subtypes for Callable would be helpful here if you know from looking at this what they are